packages/perseus-score/src/validate.ts

```diff
@@ -36,3 +36,34 @@ export function emptyWidgetsFunctional(
         return false;
     });
 }
+
+// TODO: this is very similar to emptyWidgetsFunctional,
+// they should be merged
```

Comment:
What _is_ the difference?  I can't see any difference.  It'd be good to expand this comment so folks coming to this in the future know what's involved in merging these functions.